### ISSUE TITLE
Use `initlog` and `mpi_initlog` in tests

### ISSUE THREAD

**Issue by masterleinad** [dealii/dealii#4250]
Although there are already `initlog` and `mpi_initlog` to unify the initialization of tests, a manual setup (including `std::ofstream logfile("output");`) is used in a lot of tests.

**Comment by bangerth** [comment]
Yes, that's one of those incomplete conversions that are just incredibly difficult to complete because we have sooo many tests. If someone wanted to make progress, that would be fantastic.

**Comment by bangerth** [comment]
Or if someone wanted to script this somehow.
